src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -121,7 +130,7 @@ class FormPage extends React.Component {
     const pageClasses = classNames('form-panel', route.pageConfig.pageClass);
     const data = this.formData();
 
-    if (route.pageConfig.showPagePerItem) {
+    if (route.pageConfig.showPagePerItem && !route.pageConfig.CustomPage) {
```

Comment:
Is this just for performance reasons? Looks like the only things happening in the `if` block are assigning values to `schema` & `uiSchema`, which as I understand it aren't used in a `CustomPage`.

_Rereads code before submitting review_

Oh, maybe it's because we're not expecting `route.pageConfig.arrayPath` to exist if there is a `CustomPage`?

Reply:
The array path may exist, but the schema properties may be empty, so trying to get the `.items` will result in an error (unless we optionally chain, of course). And yeah, we don't use it anyhow. :man_shrugging: